resources/css/bem/daily-challenge-date.less

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+.daily-challenge-date {
+  padding: 10px;
+  display: grid;
+  gap: 10px;
+  border-radius: @border-radius-large;
+  background-color: hsl(var(--hsl-d4));
+
+  &__header {
+    display: grid;
+    grid-template-columns: auto auto auto 150px auto auto;
```

Comment:
should be `minmax(100px, 150px)` or something to scale down to fit on narrower screens; 100px seems to fit on iphone SE screens
![image](https://github.com/user-attachments/assets/061e1b3b-57d8-4587-8cd3-ab966a83f1b1)